ERCS/erc-7584.md

```diff
@@ -0,0 +1,402 @@
+---
+eip: 7584
+title: ERC-7584 Linagee Name Registrar
+description: Defines on-chain domain names and their use for address mapping and decentralized content delivery networks (CDNs).
+author: Linagee Name Registrar (@Linagee-Name-Registrar), Mason Keresty (@mkeresty) <linagee.vision@gmail.com>, Derp Herpenstein (@DerpHerpenstein) <derp@derpfinance.com>, Chriton (@Chriton) <defisummer@gmail.com>
+discussions-to: https://ethereum-magicians.org/t/eip-linagee-name-registrar-and-ethereum-content-delivery-network/17596
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-12-27
+requires: 4804, 7584
+---
+
+## Abstract
+
+This EIP introduces a new standard that delineates on-chain permanent domain names and outlines their utilization for address mapping and their use in decentralized content delivery networks (CDNs) for [ERC-4804](./eip-4804.md).
```

Comment:
Your abstract should include a high-level technical overview of _how_ your proposal accomplishes its goals.